i18n/states/US.php

```diff
@@ -63,11 +63,5 @@
 	'WY' => __( 'Wyoming', 'woocommerce' ),
 	'AA' => __( 'Armed Forces (AA)', 'woocommerce' ),
 	'AE' => __( 'Armed Forces (AE)', 'woocommerce' ),
-	'AP' => __( 'Armed Forces (AP)', 'woocommerce' ),
-	'AS' => __( 'American Samoa', 'woocommerce' ),
-	'GU' => __( 'Guam', 'woocommerce' ),
-	'MP' => __( 'Northern Mariana Islands', 'woocommerce' ),
-	'PR' => __( 'Puerto Rico', 'woocommerce' ),
-	'UM' => __( 'US Minor Outlying Islands', 'woocommerce' ),
-	'VI' => __( 'US Virgin Islands', 'woocommerce' ),
+	'AP' => __( 'Armed Forces (AP)', 'woocommerce' )
```

Comment:
There should be a trailing comma on the last line of the array: https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#indentation